### PR TITLE
Add verify to picotool load command

### PR DESCRIPTION
**Pull request by facchinm**
This is needed to debug early flash failures

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	load := []string{filepath.Join(path, "picotool"), "load", *binary + ".uf2"}
+	load := []string{filepath.Join(path, "picotool"), "load -v", *binary + ".uf2"}
```
Comment (per1234) [inline main.go:54]:
```suggestion
	load := []string{filepath.Join(path, "picotool"), "load", "-v", *binary + ".uf2"}
```

The previous code produces an invalid command (i.e., `picotool "load -v"`):

```text
Unknown command: load -v
```

**Review by per1234 — CHANGES_REQUESTED** [review]

**Review by per1234 — CHANGES_REQUESTED** [review]
Verification should be toggled via an `rp2040load` command line flag.

The verification time might be significant with the very large programs that are possible with this board.

The Arduino boards platform framework and development software has a dedicated system for controlling the upload verification. It would be silly for Arduino to skip supporting it in the tools we created:

https://arduino.github.io/arduino-cli/dev/platform-specification/#upload-verification

**Comment by facchinm** [comment]
@per1234 you are right, the `verify` command should be added to the wrapper commandline and passed only if {verify} is specified in the core. Updating the PR right now

**Review by per1234 — APPROVED** [review]
Very nice improvement to the tool. Thanks @facchinm!

**Pull request merged**